src/UnityExtension/Assets/Editor/GitHub.Unity/UI/ProjectWindowInterface.cs

```diff
@@ -222,7 +222,8 @@ private static void OnLocksUpdate()
                 guidsLocks.Add(g);
             }
 
-            EditorApplication.RepaintProjectWindow();
+            //EditorApplication.RepaintProjectWindow();
+            UnityEditorInternal.InternalEditorUtility.RepaintAllViews();
```

Comment:
Add comment to explain the rational here...
https://github.com/github-for-unity/Unity/pull/959#discussion_r236694800